### ISSUE TITLE
Side Boarding on Rest

### ISSUE THREAD

**Issue by D-D-2** [Cards-of-Chaos/Game-Design#3]
Assuming we include the standard formula of short/short/long rests, I think it would be interesting if players could "sideboard" on a short rest. This sideboard would be 20% of the total limit of cards players can have in their deck. This way between combats, or after a combat, players can make their deck more or less social vs combat oriented as the game flows.

**Comment by nstevens26** [comment]
I like this idea a lot, to build off of it a little, rather than having the player pick a select few spells to have in their deck upon level up, they have access to all spells of their class and can change out their deck during short/long rests. Similar to how cleric, druid, and artificers operate in D&D.

**Comment by D-D-2** [comment]
Sort of like a 'soft' respec on level up of your deck. Makes sense since on level up players will be getting new spells and increased card limits.

**Comment by nstevens26** [comment]
Makes me think that each class should just have their own spell list and its up to the player how they want to build the list with said spell list.

**Comment by D-D-2** [comment]
I agree. Designing spells as only having the prereq of class and level will open the game up a lot for each player to create their own "subclass" or "archetype." I think we can design future cards with certain subclasses in mind, but keep that element implicit to allow the players to find ways to make something new from what we give them.
